unfoldingWord/translationCore
# Unmerge without empty alignment persisted

#### This pull request addresses:

- Unmerge with Blank alignment after current
- Unmerge without empty alignment persisted

#### How to test this pull request:

- Check out the same branch of the word alignment tool
- Open a project to align AFTER deleting the existing alignment data
- Drag and drop empty greek words to conjoin
- Drag and drop a conjoined greek word to the new empty alignment that pops up
- Do the same after dropping word bank words into each
- Ensure all functions as expected


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2872)
